### PR TITLE
Fix/update tool handling

### PR DESCRIPTION
**Pull request by jabou**
This PR includes:
- Podspec fixes (removed unused tools, added Collar, removed Loggie version lock)
- Added preselection of Tools tab
- Fixed issue with deselecting tool cell when tool is presented modally

Additionally, min supported iOS version is raised from iOS 10 to iOS 11.

I'm proposing setting the next version as 1.1.0 as we introduced a new tool and added a new visually visible change (tool tab preselected).

_I didn't update the readme file as this is something prepared for the next batch of internal tools updates and the current readme should be updated extensively._

### CODE REVIEW

File: Example/Sentinel/AppDelegate.swift [hunk 1]
```diff
@@ -39,10 +39,9 @@ private extension AppDelegate {
             tools: [
                 UserDefaultsTool(),
                 baseUrlTool,
-                CustomLocationTool(),
-//                BugsnatchTool(triggerActionConfig: EmailConfig()),
-//                LoggieTool(),
-//                AnalyticsCollectorTool(),
+                CustomLocationTool()
+//                CollarTool(),
```
Comment (hhrvoic) [inline Example/Sentinel/AppDelegate.swift:43]: I think we can delete these comments

File: Sentinel/Classes/Core/Internal/SentinelTabBarController.swift [hunk 2]
```diff
@@ -28,3 +36,15 @@ extension SentinelTabBarController {
...
+
+// MARK: - Private methods -
+
+private extension SentinelTabBarController {
+
+    func setPreselectedActionIfNeeded() {
+        defer { didPreselectAction = true }
+        guard !didPreselectAction else { return }
+        // Preselect Tools tab
+        selectedIndex = 2
```
Comment (hhrvoic) [inline Sentinel/Classes/Core/Internal/SentinelTabBarController.swift:48]: Would rather use enum for tab screens with index as raw value if possible, you wouldn't need comment above then :)

**Review by hhrvoic — APPROVED** [review]
Looks good, minor comments added

File: Example/Sentinel/AppDelegate.swift (hunk 1 above)
Comment (jabou) [reply Example/Sentinel/AppDelegate.swift:43]: We left them in the example project just for the sake of example on how you can add collar and analytics tools if you want to include them.

File: Sentinel/Classes/Core/Internal/SentinelTabBarController.swift (hunk 2 above)
Comment (jabou) [reply Sentinel/Classes/Core/Internal/SentinelTabBarController.swift:48]: Let's leave it for now, but I'll open todo on Productive for this to be handled in the future :)

**Pull request merged**